src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -296,6 +302,7 @@ export const SueReportLocation = ({
       <WrapperHorizontal>
         <Map
           calloutTextEnabled
+          clusterDistance={configuration.geoMap?.clusterDistance}
```

Comment:
thats all? wow. and the given value per configuration is the correct one that the map understands?

Reply:
Yes. The value comes as the map wants. 👍